### PR TITLE
Add script describing how the gatk/germline standard germline variant…

### LINKED ISSUE #391: Document test BAMs and VCF files
@timodonnell Do you have any documentation for how `https://github.com/hammerlab/guacamole/tree/master/src/test/resources/illumina-platinum-na12878` was created? Specifically, the commands for the other VCF files?

### PR DESCRIPTION
**Pull request by timodonnell**
… callers were run to generate test vcfs. Fixes #391

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/guacamole/392)

### CODE REVIEW

File: src/test/resources/illumina-platinum-na12878/run_other_callers.readme.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Change this to point to your GATK jar.
+GATK_JAR=~/oss/gatk-binary/GenomeAnalysisTK.jar
+
+# Unified genotyper
+time java -jar $GATK_JAR \
+    -T UnifiedGenotyper \
+    -R src/test/resources/illumina-platinum-na12878-extract/chr1.prefix.fa \
+    -I src/test/resources/illumina-platinum-na12878-extract/NA12878.10k_variants.plus_chr1_3M-3.1M.chr_fixed.bam \
+    -L chr1:1-6700000 \
+    -glm INDEL \
```
Comment (arahuja) [inline src/test/resources/illumina-platinum-na12878/run_other_callers.readme.sh]: Should this be 'BOTH'? Is this why UnifiedGenotyper had ~8 % sensitivity in our test?

Comment (timodonnell) [inline src/test/resources/illumina-platinum-na12878/run_other_callers.readme.sh]: Good catch -- I think I had switched this and then forgot to put it back

Comment (timodonnell) [inline src/test/resources/illumina-platinum-na12878/run_other_callers.readme.sh]: I think there's some other issues too with paths, I'll fix this up

**Comment by timodonnell** [comment]
Ok, fixed the paths in the script and changed the unified genotyper run to call both indels and snvs. This should be good to merge

**Comment by arahuja** [comment]
Great, thanks @timodonnell!

**Pull request merged**